ValetTests/ValetTests.m

```diff
@@ -174,20 +174,20 @@ - (void)test_stringForKey_differentIdentifierFailsToRetrieveString;
     XCTAssertNil(string, @"Expected string with Key with different identifier to be nil but instead it was %@", string);
 }
 
-- (void)test_stringForKey_differentAccessGroupFailsToRetrieveString;
+- (void)test_stringForKey_differentAccessibilityFailsToRetrieveString;
 {
     XCTAssertTrue([self.valet setString:self.string forKey:self.key]);
     VALValet *otherValet = [[VALValet alloc] initWithIdentifier:self.valet.identifier accessibility:VALAccessibilityAfterFirstUnlockThisDeviceOnly];
     [self.additionalValets addObject:otherValet];
     
     NSString *string = [otherValet stringForKey:self.key];
-    XCTAssertNil(string, @"Expected string with Key with different access group to be nil but instead it was %@", string);
+    XCTAssertNil(string, @"Expected string with Key with different accessibility to be nil but instead it was %@", string);
 }
 
 - (void)test_stringForKey_differentValetTypeFailsToRetrieveString;
```

Comment:
Actually, I can just remove the test once I merge this. Thanks again for helping clean up our tests!